### PR TITLE
Add document for default block size

### PR DESCRIPTION
**Pull request by halfcrazy**
https://github.com/projectcalico/node/pull/339

## Todos

- [ ] Tests
- [x] Documentation
- [x] Release note

## Release Note

```release-note
Add document for default block size configuration
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 572f34b72ba14fb1b3854d024c80b627b189121e

https://deploy-preview-2920--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 5a4ad6f85662aa809c7ee9144c6f9001ec30910b

https://deploy-preview-2920--calico.netlify.com

File: master/reference/node/configuration.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ The `{{site.nodecontainer}}` container is primarily configured through environme
 | CALICO_NETWORKING_BACKEND | The networking backend to use.  In `bird` mode, Calico will provide BGP networking using the BIRD BGP daemon; VXLAN networking can also be used.  In `vxlan` mode, only VXLAN networking is provided; BIRD and BGP are disabled.  If set to `none` (also known as policy-only mode), both BIRD and VXLAN are disabled. [Default: `bird`] | bird, vxlan, none |
 | CALICO_IPV4POOL_CIDR | The IPv4 Pool to create if none exists at start up. It is invalid to define this variable and NO_DEFAULT_POOLS. [Default: `192.168.0.0/16`] | IPv4 CIDR |
 | CALICO_IPV6POOL_CIDR | The IPv6 Pool to create if none exists at start up. It is invalid to define this variable and NO_DEFAULT_POOLS. [Default: `<a randomly chosen /48 ULA>`] | IPv6 CIDR |
+| CALICO_IPV4POOL_BLOCK_SIZE | Block size to use for the IPv4 POOL created at start up.  Block size for IPv4 should in range 20~32 (inclusive) [Default: `26`] | int |
```
Comment (bmckercher123) [inline master/reference/node/configuration.md]: CALICO_IPV4POOL_BLOCK_SIZE | Block size to use for the IPv4 POOL created at startup.  Block size for IPv4 should in range 20-32 (inclusive).

(changed "startup" and "20-32")

File: master/reference/node/configuration.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ The `{{site.nodecontainer}}` container is primarily configured through environme
...
+| CALICO_IPV6POOL_BLOCK_SIZE | Block size to use for the IPv6 POOL created at start up.  Block size for IPv6 should in range 116~128 (inclusive) [Default: `122`] | int |
```
Comment (bmckercher123) [inline master/reference/node/configuration.md]: CALICO_IPV6POOL_BLOCK_SIZE | Block size to use for the IPv6 POOL created at startup.  Block size for IPv6 should in range 116-128 (inclusive)

(changed "startup" and "20-32")

File: master/reference/node/configuration.md (hunk 1 above)
Comment (tmjd) [reply master/reference/node/configuration.md]: I think `should in range` needs to be changed too. what about `should be in the range`?

Comment (halfcrazy) [reply master/reference/node/configuration.md]: done

File: master/reference/node/configuration.md (hunk 2 above)
Comment (halfcrazy) [reply master/reference/node/configuration.md]: done

**Comment by tmjd** [comment]
I'm closing and will reopen to try to get the license/cla to work.

**Pull request merged**